static/styles/zulip.css

```diff
@@ -1577,6 +1577,7 @@ div.focused_table {
         margin: 0 -4px 0 0;
         padding: 12px 6px;
         overflow: hidden;
+        text-overflow: ellipsis;
```

Comment:
Same question with this -- it feels odd that we have this set on `.stream,` rather than just a generic rule on the parent.

Reply:
This was just a style choice, we can have the `.stream` overflow with ellipses rather than just clipping. I'll make that change.